base/string.jl

```diff
@@ -968,8 +968,8 @@ byte_string_classify(s::ByteString) = byte_string_classify(s.data)
     # 1: valid ASCII
     # 2: valid UTF-8
 
-is_valid_ascii(s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) == 1
-is_valid_utf8(s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) != 0
+isvalid(::Type{ASCIIString}, s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) == 1
```

Comment:
I've never made an ultimata.  Please stop describing what I said as such.

I was simply responding to this:
>Although I don't think you should wait on #11004 for other improvements.

It's simply hard to justify spending more time improving other parts of Julia, when I haven't gotten any movement on improving the part that is most critical to my work.
If you want outside people to freely contribute their time and effort to the success of Julia, and in turn Julia Computing, I think it is good to be responsive to their needs occasionally also.

Also:
>Code in Base has no special performance privileges.

That's not really true, because there a lot of things that are doing conversions under the hood,
such as calling Windows Unicode APIs, or with the ODBC package, etc.  Unless the performance
issues are fixed in Base, adding a "ScottStrings" module won't help at all...